### DISCUSSION TITLE
collector architecture

### DISCUSSION THREAD

**Discussion by SyntaxEvg** [open-telemetry/opentelemetry-collector-contrib#19148] [General]
### Describe the issue you're reporting

hello, I'm trying to study the collector architecture and draw a diagram of how everything works here, but there are many different options on the Internet, including "Agents", do you have a detailed diagram of how these components are connected to each other
this option brought me to a stupor, why is the pipeline repeated here, above and below, can you answer this question for me?
![image](https://user-images.githubusercontent.com/94603687/221402846-d7972c7b-a578-436a-88a6-bbbfb7ecf159.png)

**Comment by jpkrohling** [comment]
Agent is an unfortunate name used when deploying an OTel Collector close to your workloads, like when it's deployed as a daemonset of sidecar. The binary is the same, with the possible difference that the expectation of an agent is to receive data from workloads and forward it to centralized collectors.

I recommend this page, which contains an architecture diagram: https://opentelemetry.io/docs/collector/

@mhausenblas is also building a documentation page with use-cases and patterns for plugging collectors together. Perhaps he can share an early draft with you.

**Comment by mhausenblas** [comment]
Thanks @jpkrohling and the respective PR is https://github.com/open-telemetry/opentelemetry.io/pull/2312 with a [preview](https://deploy-preview-2312--opentelemetry.netlify.app/docs/collector/deployment/) available as well.
